2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,8 +18,14 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
-}
+let currencyCodes = {};
+countryCurrencyCodes.forEach((element) => 
+currencyCodes[0] = element[1]
```

Comment:
Here is a small thing to fix the end of the entire exercise will be done perfectly! 👍 
Please think over where You can use bracket notation in the line 23.